interpreters/toywasm/Makefile

```diff
@@ -51,21 +53,17 @@ CSRCS += vec.c
 CSRCS += wasi.c
 CSRCS += xlog.c
 
-CFLAGS += -DTOYWASM_USE_SEPARATE_EXECUTE
-CFLAGS += -DTOYWASM_USE_TAILCALL
-CFLAGS += -DTOYWASM_USE_JUMP_BINARY_SEARCH
-CFLAGS += -DTOYWASM_JUMP_CACHE2_SIZE=4
-CFLAGS += -DTOYWASM_USE_SEPARATE_LOCALS
-CFLAGS += -DTOYWASM_USE_SMALL_CELLS
-CFLAGS += -DTOYWASM_ENABLE_WRITER
+CFLAGS += ${shell $(INCDIR) "$(CC)" $(APPDIR)/interpreters/toywasm/include}
```

Comment:
why need this path? I don't find it.

Reply:
to include this flie. https://github.com/apache/incubator-nuttx-apps/blob/16e873587ca898e66e749a22c793dfbcdb919515/interpreters/toywasm/include/toywasm_config.h

(in upstream toywasm repo, this header file is generated by cmake. for nuttx apps, i simply added the generated file instead.)

Reply:
Ok, I see.